wire-ios-sync-engine/Source/Synchronization/Strategies/CallingRequestStrategy.swift

```diff
@@ -332,22 +336,38 @@ extension CallingRequestStrategy: WireCallCenterTransport {
 
             self.zmLog.debug("schedule calling message")
 
-            let genericMessage = GenericMessage(content: Calling(content: dataString))
             let recipients = targets.map { self.recipients(for: $0, in: self.managedObjectContext) } ?? .conversationParticipants
 
-            let message = GenericMessageEntity(
-                conversation: conversation,
-                message: genericMessage,
-                targetRecipients: recipients,
-                completionHandler: nil
-            )
+            let message: GenericMessageEntity
+
+            if overMLSSelfConversation, conversation.messageProtocol == .mls {
+                guard let selfConversation = ZMConversation.fetchSelfMLSConversation(in: self.managedObjectContext) else {
+                    WireLogger.mls.error("missing self conversation for sending message to own clients")
+                    completionHandler(500)
+                    return
+                }
+
+                message = GenericMessageEntity(conversation: selfConversation,
+                                               message: genericMessage,
+                                               targetRecipients: recipients,
+                                               completionHandler: nil)
+            } else {
+                message = GenericMessageEntity(
+                    conversation: conversation,
+                    message: genericMessage,
+                    targetRecipients: recipients,
+                    completionHandler: nil
+                )
+            }
 
             switch (conversation.messageProtocol, recipients) {
             case (.proteus, _), (.mls, .conversationParticipants):
                 message.send(with: self.messageSync, completion: completionHandler)
 
             case (.mls, _):
-                if message.isConferenceKey {
+                // TODO: review the `isConferenceKey` case once subconversations are available
+                // to target all conference members
+                if message.isConferenceKey || overMLSSelfConversation {
                     message.send(with: self.messageSync, completion: completionHandler)
                 } else {
                     Logging.mls.info("ignoring targeted outgoing calling message b/c its not CONFKEY")
```

Comment:
I guess this message is outdated since we'd also be sending targeted messages if `overMLSSelfConversation == true`

Reply:
@David-Henner do you have a suggestion of what it should be?

Is that sufficient?
```suggestion
                    Logging.mls.info("ignoring message using mls")
```

Reply:
I was thinking of something like that `"ignoring targeted outgoing calling message b/c it's not CONFKEY nor sent over self conversation"`